static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -115,11 +118,30 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 			var originalFile = resultObject.getDatastream("original_file");
 			items["viewFile"] = {name : "View File"
 				+ " ("+ StringUtilities.readableFileSize(originalFile['fileSize']) + ")"};
-
-			items["viewFits"] = {name: "View FITS"}
 		}
-		if ($.inArray('viewHidden', metadata.permissions) != -1) {
-			items["viewEventLog"] = {name : "View Event Log"};
+
+		if (viewHidden || isFile || /md_descriptive|tech_md/ig.test(datastreams)) {
+			items["metadata"] = {name: "View metadata", items: {}}
+
+			if (/md_descriptive/ig.test(datastreams)) {
+				items['metadata']['items']["viewMods"] = {name: "View MODS"};
+			}
+
+			if (/md_descriptive_history/ig.test(datastreams)) {
```

Comment:
this datastream requires `viewHidden` permission, at the moment i can see the option for items that I can't retrieve the datastream for (gives a blank screen). I tested this by spoofing as an observer and then checked the available options for Units I didn't have admin permission to.

Reply:
That might explain Anna's error. :)